src/index.ts

```diff
@@ -1,182 +1,184 @@
-// Colors

```

Comment:
why has this file changed?
crlf issue?

Reply:
No idea, editor changed it

Reply:
![image](https://user-images.githubusercontent.com/6171742/61543924-ad176380-aa44-11e9-806c-4b35789a0003.png)